packages/slate-tables/src/normalization/insertMissingCells.ts

```diff
@@ -12,15 +12,9 @@ export function insertMissingCells(editor: TablesEditor, path: Path) {
         return false;
     }
 
-    const maxWidth = table.children.reduce((max, row) => {
-        const rowSize = calculateRowWidth(row);
```

Comment:
This line was assuming `table.children` are always `TableRowNode[]`. But sometimes it's not.